### DISCUSSION TITLE
Incremental update on Minio

### DISCUSSION THREAD

**Discussion by alltheerrorsknowme** [minio/minio#19011] [Q&A] [answered]
Hi, I have a question that I have been struggling with. I am going to describe it. I have tables in MSSQL,  I needed to migrate them to the MinIO object storage and I succeeded to do it. At the end all the files at the bucket is 0.5 GB which is good. Files are stored in the form of Parquet files. Now challenge is that, How can I update only incremental changes to these files from database (like lets say one of the values got updated and I want this change to be reflected on Minio as well). As far as I know, files on Minio are immutable so you can not update them. Is there anyone that can give recommendations. Thanks in advance

**Comment by klauspost** [comment, marked as answer]
As you point out there no S3 API for partially updating objects. So you will need a different approach.
